2017-04-14 01:00:07.3089|ERROR|Cafe2u.Application.WebApi.Helpers.NlogExceptionLogger|Request: http://localhost:12345/img/favicon.png
Unhandled exception: System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.HttpListenerRequest'.
   at System.Net.HttpListenerRequest.CheckDisposed()
   at System.Net.HttpListenerRequest.get_LocalEndPoint()
   at System.Net.HttpListenerRequest.get_IsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue(String key, Object& value)
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue(String key, Object& value)
   at Microsoft.Owin.OwinContext.Get[T](String key)
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal()
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail()
   at System.Net.Http.HttpRequestMessageExtensions.ShouldIncludeErrorDetail(HttpRequestMessage request)
   at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse(HttpRequestMessage request, HttpStatusCode statusCode, Func`2 errorCreator)
   at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse(HttpRequestMessage request, HttpStatusCode statusCode, String message, String messageDetail)
   at System.Web.Http.Dispatcher.HttpRoutingDispatcher.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DelegatingHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at Autofac.Integration.WebApi.Owin.DependencyScopeHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.DelegatingHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
2017-04-14 01:00:07.3779|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.HttpListenerRequest'.
   at System.Net.HttpListenerRequest.CheckDisposed()
   at System.Net.HttpListenerRequest.get_LocalEndPoint()
   at System.Net.HttpListenerRequest.get_IsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue(String key, Object& value)
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue(String key, Object& value)
   at Microsoft.Owin.OwinContext.Get[T](String key)
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal()
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail()
   at System.Web.Http.ExceptionHandling.LastChanceExceptionHandler.CreateDefaultLastChanceResult(ExceptionContext context)
   at System.Web.Http.ExceptionHandling.LastChanceExceptionHandler.HandleAsync(ExceptionHandlerContext context, CancellationToken cancellationToken)
   at System.Web.Http.ExceptionHandling.ExceptionHandlerExtensions.<HandleAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<InvokeCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Owin.AutofacAppBuilderExtensions.<>c__DisplayClass8_0.<<RegisterAutofacLifetimeScopeInjector>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
2017-04-14 01:09:14.0413|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:09:14.0413|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:09:14.0413|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:09:14.0413|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:24:04.1519|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:48:54.3198|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 01:48:54.3238|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 13:50:55.9726|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 13:50:55.9726|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 13:53:10.9948|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 13:53:10.9968|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 14:40:51.5951|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 14:40:51.5951|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 14:52:34.0983|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.OperationCanceledException: The operation was canceled.
   at System.Threading.CancellationToken.ThrowOperationCanceledException()
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<BufferResponseContentAsync>d__13.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<InvokeCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Owin.AutofacAppBuilderExtensions.<>c__DisplayClass8_0.<<RegisterAutofacLifetimeScopeInjector>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
2017-04-14 14:54:41.8767|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 15:08:57.3010|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 15:08:57.3010|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 15:09:02.3046|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-14 15:18:22.1184|ERROR|Cafe2u.Application.WebApi.Helpers.NlogExceptionLogger|Request: http://localhost:12345/api/Users/GetUsers?isHidden=0&from=0&count=30&sortBy=id&IsAscending=true&_=1492172301313
Unhandled exception: System.Net.Http.HttpRequestException: Error while copying content to a stream. ---> System.IO.IOException ---> System.Net.HttpListenerException: The parameter is incorrect
   at System.Net.HttpResponseStream.BeginWrite(Byte[] buffer, Int32 offset, Int32 size, AsyncCallback callback, Object state)
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.ExceptionFilterStream.BeginWrite(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   --- End of inner exception stack trace ---
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.ExceptionFilterStream.BeginWrite(Byte[] buffer, Int32 offset, Int32 count, AsyncCallback callback, Object state)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncImpl[TArg1,TArg2,TArg3](Func`6 beginMethod, Func`2 endFunction, Action`1 endAction, TArg1 arg1, TArg2 arg2, TArg3 arg3, Object state, TaskCreationOptions creationOptions)
   at System.Net.Http.HttpContent.CopyToAsync(Stream stream, TransportContext context)
   --- End of inner exception stack trace ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<SendResponseContentAsync>d__20.MoveNext()
2017-04-14 15:18:22.1344|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<SendResponseContentAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<InvokeCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Owin.AutofacAppBuilderExtensions.<>c__DisplayClass8_0.<<RegisterAutofacLifetimeScopeInjector>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
2017-04-14 15:20:27.4921|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
